windows/Clipboard/Clipboard.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#include "pch.h"
+
+#include <functional>
+#include <string>
+#include <NativeModules.h>
+#include <../Common/unicode.h>

```

Comment:
Where is this being pulled from when we're in a real app? This seems like something kind of private that may be removed or moved around in RNW.

Reply:
reworking the native module build logic is outside the scope of this PR, feel free to file an issue if you'd like.

Reply:
I'm referring specifically to the Unicode header which we don't need here.